src/codegen.cpp

```diff
@@ -8209,42 +8209,20 @@ extern "C" void jl_init_llvm(void)
 #endif
 
     // Parse command line flags after initialization
-    const char *const argv_tailmerge[] = {"", "-enable-tail-merge=0"}; // NOO TOUCHIE; NO TOUCH! See #922
-    cl::ParseCommandLineOptions(sizeof(argv_tailmerge)/sizeof(argv_tailmerge[0]), argv_tailmerge, "disable-tail-merge\n");
-#if defined(_OS_WINDOWS_) && defined(_CPU_X86_64_)
-    const char *const argv_copyprop[] = {"", "-disable-copyprop"}; // llvm bug 21743
-    cl::ParseCommandLineOptions(sizeof(argv_copyprop)/sizeof(argv_copyprop[0]), argv_copyprop, "disable-copyprop\n");
```

Comment:
https://bugs.llvm.org/show_bug.cgi?id=21743 fixed by
```
commit 796d906e06f8f5a3c8d4f5f0faa5c47697153490
Author: Quentin Colombet <qcolombet@apple.com>
Date:   Thu Apr 23 21:17:39 2015 +0000

    [MachineCopyPropagation] Handle undef flags conservatively so that we do not
    remove copies that are useful after breaking some hardware dependencies.
    In other words, handle this kind of situations conservatively by assuming reg2
    is redefined by the undef flag.
    reg1 = copy reg2
    = inst reg2<undef>
    reg2 = copy reg1
    Copy propagation used to remove the last copy.
    This is incorrect because the undef flag on reg2 in inst, allows next
    passes to put whatever trashed value in reg2 that may help.
    In practice we end up with this code:
    reg1 = copy reg2
    reg2 = 0
    = inst reg2<undef>
    reg2 = copy reg1
    
    This fixes PR21743.
    
    llvm-svn: 235647
```